src/OpenTelemetry.ResourceDetectors.Azure/AppServiceResourceDetector.cs

```diff
@@ -0,0 +1,59 @@
+// <copyright file="AppServiceResourceDetector.cs" company="OpenTelemetry Authors">
```

Comment:
Other projects in this contrib repo has placed this files in Resources folder. Is there a convention followed here?

Reply:
I think that it is leftover, from old package name, I would expect have it directly in project folder in this case.